bin/tronctl

```diff
@@ -103,7 +103,7 @@ def parse_date(date_string):
 def parse_cli():
     parser = cmd_utils.build_option_parser()
     subparsers = parser.add_subparsers(dest="command", title="commands", help="Tronctl command to run")
-    subparsers.required = True  # add_subparsers only supports required arg from py37
+    subparsers.required = True  # add_subparsers only supports required arg from py38
```

Comment:
```python
subparsers = parser.add_subparsers(dest="command", title="commands", help="Tronctl command to run", required=True)
``` 
should work here now that we're gonna be running on 3.7+ :)

(this comment was referring to in `required - Whether or not a subcommand must be provided, by default False (added in 3.7)` https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_subparsers